src/server/service/kv.rs

```diff
@@ -1558,18 +1558,26 @@ fn future_raw_batch_put<E: Engine, L: LockManager>(
     mut req: RawBatchPutRequest,
 ) -> impl Future<Output = ServerResult<RawBatchPutResponse>> {
     let cf = req.take_cf();
+    let pairs_len = req.get_pairs().len();
     let pairs = req
         .take_pairs()
         .into_iter()
         .map(|mut x| (x.take_key(), x.take_value()))
         .collect();
+    let ttls = if req.get_ttls().is_empty() {
+        vec![0; pairs_len]
+    } else if req.get_ttls().len() == 1 {
+        vec![req.get_ttls()[0]; pairs_len]
+    } else {
+        req.take_ttls()
+    };
```

Comment:
Please add a comment describing the reason of this behavior (the historical behavior, the latter change)